src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/custom.py

```diff
@@ -4,6 +4,11 @@
 # --------------------------------------------------------------------------------------------
 import time
 import uuid
+# to ensure we have consistent access to the quote() method
+try:
+    from urllib import parse as safe_url_lib
+except ImportError:
+    import urllib as safe_url_lib
```

Comment:
The `six` compatibility library should handle this.
We use this elsewhere in the CLI.
https://pythonhosted.org/six/#module-six.moves.urllib.parse